apps/main/src/lend/components/PageLoanManage/Page.tsx

```diff
@@ -67,6 +69,23 @@ const Page = () => {
     llamma: market,
     llammaId: rOwmId,
   })
+  const userCollateralEvents = useUserLendCollateralEvents({
+    chainId: rChainId,
+    controllerAddress: market?.addresses?.controller || '',
+    userAddress: signerAddress || '',
+    collateralToken: market?.collateral_token
+      ? {
+          symbol: market.collateral_token.symbol || '',
+          address: market.collateral_token.address || '',
+        }
+      : { symbol: '', address: '' },
+    borrowToken: market?.borrowed_token
+      ? {
+          symbol: market.borrowed_token.symbol || '',
+          address: market.borrowed_token.address || '',
+        }
+      : { symbol: '', address: '' },
+  })
```

Comment:
Queries should not need '' as default

```suggestion
    controllerAddress: market?.addresses?.controller,
    userAddress: signerAddress,
    collateralToken: market?.collateral_token,
    borrowToken: market?.borrowed_token,
  })
```